server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/IpTermsIT.java

```diff
@@ -140,22 +140,24 @@ public void testMissingValue() throws Exception {
             client().prepareIndex("index").setId("3").setSource("ip", "127.0.0.1"),
             client().prepareIndex("index").setId("4").setSource("not_ip", "something")
         );
-        SearchResponse response = client().prepareSearch("index")
-            .addAggregation(AggregationBuilders.terms("my_terms").field("ip").missing("127.0.0.1").executionHint(randomExecutionHint()))
-            .get();
-
-        assertSearchResponse(response);
-        Terms terms = response.getAggregations().get("my_terms");
-        assertEquals(2, terms.getBuckets().size());
-
-        Terms.Bucket bucket1 = terms.getBuckets().get(0);
-        assertEquals(2, bucket1.getDocCount());
-        assertEquals("127.0.0.1", bucket1.getKey());
-        assertEquals("127.0.0.1", bucket1.getKeyAsString());
-
-        Terms.Bucket bucket2 = terms.getBuckets().get(1);
-        assertEquals(2, bucket2.getDocCount());
-        assertEquals("192.168.1.7", bucket2.getKey());
-        assertEquals("192.168.1.7", bucket2.getKeyAsString());
+        assertBusy(() -> {
```

Comment:
There is one more test failure `testScriptValue` in this class. Can we also fix that ?